### ISSUE TITLE
Clonar por https

### ISSUE THREAD

**Issue by PalumboN** [faloi/yanapiri#46]
Actualmente al clonar los repos se hace por ssh, pero algunas facus tienen los puertos bloqueados y no se puede resolver el host:
```
ssh: connect to host github.com port 22: No route to host
fatal: Could not read from remote repository.
```
(Al conectarme desde mi cel me deja clonar sin problemas).

Estaría bueno tener la opción o de alguna manera poder clonar los repos por https.

**Comment by faloi** [comment]
Sí, habría que agregar una opción, aunque no sé cómo se manejaría la cuestión de pedir el password mientras el CLI se ejecuta.

Probá este truco mientras tanto, a mí me funcionó varias veces que me pasó eso mismo: https://help.github.com/en/github/authenticating-to-github/using-ssh-over-the-https-port.
